app/assets/stylesheets/notice.scss

```diff
@@ -0,0 +1,17 @@
+.notice {
+  background-color: #D2D6DB; // smoke-20
+  border-radius: 3px;
+  border: 0.5px solid #99A3AD; // smoke-40
```

Comment:
I'm not sure I'm sold on using 0.5px units - these look sharp and crispy on retinas, but blurry on non-retinas.

Reply:
I'm going to get rid of this file and use the styles in `components/notices.scss` :)